src/App.js

```diff
@@ -12,12 +12,62 @@ import JournalForm from './Journal/JournalForm'
 import JournalList from './Journal/JournalList'
 import { useState } from 'react'
 import { loadLocally, saveLocally } from './Journal/util'
+import Header from './common/Header'
 
 export default function App() {
   const [journalEntries, setJournalEntries] = useState(
     loadLocally('journalEntries') ?? []
   )
 
+  useEffect(() => saveLocally('journalEntries', journalEntries), [
+    journalEntries,
+  ])
```

Comment:
You could move this easily into a custom hook, your App.js looks a bit cleaner this way. Would also move line 71-82 in the same custom hook and call it useJournalEntries.

Reply:
Good idea! I added the custom hook!